src/Razor/src/Microsoft.VisualStudio.LanguageServerClient.Razor/DefaultLSPEditorFeatureDetector.cs

```diff
@@ -162,12 +161,8 @@ private protected virtual bool EnvironmentFeatureEnabled()
         // Private protected virtual for testing
         private protected virtual bool IsFeatureFlagEnabledCached()
         {
-            if (!_featureFlagEnabled.HasValue)
-            {
-                _featureFlagEnabled = _featureFlags.IsFeatureEnabled(RazorLSPEditorFeatureFlag, defaultValue: false);
-            }
-
-            return _featureFlagEnabled.Value;
+            var featureFlagEnabled = _featureFlags.IsFeatureEnabled(RazorLSPEditorFeatureFlag, defaultValue: false);
+            return featureFlagEnabled;
```

Comment:
Does the feature flag text need to change to no longer indicate that VS needs to restart if it's changed?

Reply:
I kept that in as I think we may be depending on a restart in other areas. 

The text we had before (https://github.com/dotnet/aspnetcore-tooling/pull/2037/files#diff-18b9b146f4d27a6a5b6248ece31b30570d84c2a409b25bf5b660de86e608fa4cL15) was pretty verbose too. I think just a `restart` is more straightforward.

Reply:
Ya there's a few other cases where we require restart

Reply:
OK, I find the previous text to be more correct now, but yeah, it was more verbose.